### ISSUE TITLE
Pixel and vertex shader code does not contain latest Khronos PBR fixes

### ISSUE THREAD

**Issue by ziriax** [Microsoft/glTF-DXViewer#1]
First of all many thanks for this sample! 

At [Wonder Media](http://wondermedia.tv), we are using DirectX 11 for our custom render and animation engine that uses glTF as model files, and we also made the open source [Maya2glTF](https://github.com/Ziriax/Maya2glTF)  exporter. It's nice to see an open source glTF viewer based on DX11!

I briefly looked at the code and it seems your shaders are a bit older than the one in the Khronos PBR repos.

Specifically, [my pull request](https://github.com/KhronosGroup/glTF-WebGL-PBR/pull/69) that fixed shading non-uniformly scaled models is missing, and also [this pull request](https://github.com/KhronosGroup/glTF-WebGL-PBR/pull/80) that fixed a bug where NdotV can be >1.0.

**Comment by peted70** [comment]
Yes, thanks for bringing attention to that. We should at least keep up with any fixes/improvements there and possibly add other shader options moving forwards.
